spring-kafka/src/test/java/org/springframework/kafka/support/DefaultKafkaHeaderMapperTests.java

```diff
@@ -378,6 +522,19 @@ void ensureNullHeaderValueHandledGraciously() {
 		verify(mockHeader, never()).key();
 	}
 
+	private List<byte[]> extractHeaderValues(Headers headers, String headerName) {
+		return StreamSupport.stream(headers.headers(headerName).spliterator(), false)
+							.map(Header::value)
+							.toList();
```

Comment:
I believe AssertJ can do assertion not only on List. I mean with AssertJ API it might turn out we don’t need this method.